dom.bs

```diff
@@ -3594,6 +3625,60 @@ run these steps:
 <ol>
  <li><p>Assert: either <var>addedNodes</var> or <var>removedNodes</var> <a for=set>is not empty</a>.
 
+ <li><p>If either <var>options</var>'s' {{MutationObserverInit/elementLocalNameFilter}} or
+ {{MutationObserverInit/elementHasAttributeNameFilter}} is present, then:
+
+ <ol>
+  <li><p>Let <var>nodeAttributesList</var> be <var>nodeAttributesList</var> of the node's attributes, "<code>node.getAttributeNames()</code>".
```

Comment:
This isn't right. node.getAttributeNames returns a list of qualified names.
In general, we don't want to be replying on other DOM APIs to define these algorithmic steps.
Also, node's attribute has a bunch of things: https://dom.spec.whatwg.org/#concept-attribute
What we want here is the local name so either we pick that here, or reference it when we filter it.

Reply:
Is this correct? Later it is used to check if node's attributes contains any attributes from the `elementFilterByAttribute` list. If `nodeAttributesList` is only the local name of the node, it cannot filter properly. Maybe I'm mistaken?

https://github.com/whatwg/dom/pull/885/files/7eb45a14fff8de0268e64bd2787b8d7f08357c69#diff-a467e681501e456ce8c1ef31425e1b41R3651

Reply:
@rniwa - would the following suffice putting it inside of a code block? Or how is it preferred to be written here?
```
<code>
var nodeAttributesList = [];
for (var i = 0; i < element.attributes.length; i ++) { 
    nodeAttributesList.push(element.attributes[i].localName);
}
</code>
```

Reply:
No, these DOM normative text shouldn't be written as JavaScript or invoking other DOM API. We should probably just say Let nodeAttributesList be the set of local names of node's attributes.

Reply:
Works for me 👍